StRoot/StMcEventMaker/StMcEventMaker.cxx

```diff
@@ -621,7 +625,7 @@ Int_t StMcEventMaker::Make()
 	    mtdHitTable = g2t_mtd_hitTablePointer->GetTable();
 	else
 	    if (Debug()) cerr << "Table g2t_mtd_hit Not found in Dataset " << geantDstI.Pwd()->GetName() << endl;
-
+#if 0
```

Comment:
Similar to the discussion in PR#281, Xianglei, does this need to be changed to #if 1 or remove completely? Let us resolve this soon. Thanks

Reply:
Xin, it is fine for me to keep it as it is.

Reply:
Xianglei, I don't quite understand. With "#if 0", we will be skipping all the BEMC, FCS hits? I would suggest to remove it then?

Reply:
oh, I did not realize this, please go ahead to remove it then...